sleap/gui/dialogs/importvideos.py

```diff
@@ -601,8 +619,11 @@ def plot(self, idx=0):
         frame = self.video.get_frame(idx)
         # Clear existing objects
         self.view.clear()
+        # Re-size the preview image
+        decimate = 10
+        frame_resized = frame[::decimate, ::decimate]
```

Comment:
Test what happens when we decimate a small image s.t. the resized image is smaller than the image being displayed.